library/lib/hot-css-replacement-client.js

```diff
@@ -16,10 +16,16 @@ function hotCssReplacementClient(port, cssHash, chunkName) { // eslint-disable-l
 
     switch (type) {
       case 'done':
-        const cssHash = cssChunkHashes[chunkName]
-        if (!cssHash || (previousCssHash === cssHash)) return
-        document.querySelector(`link[href="/${previousCssHash}.css"]`).setAttribute('href', `/${cssHash}.css`)
-        previousCssHash = cssHash
+        const cssHashes = cssChunkHashes[chunkName] || []
+
+        cssHashes.forEach((cssHash, index) => {
+          const previousCssHash = previousCssHashes[index]
+          if (!cssHash || (previousCssHash === cssHash)) return
+          document.querySelector(`link[href="/${previousCssHash}.css"]`).setAttribute('href', `/${cssHash}.css`)
```

Comment:
Shouldn't we use `__webpack_public_path__` instead of the leading slash?

Reply:
We need to pass it in though, since this piece of script is inlined it will not have any webpack references.